### ISSUE TITLE
[Submit add-on]: TeleNVDA 2026.7.18

### ISSUE THREAD

**Issue by jmdaweb** [nvaccess/addon-datastore#10298]
### Download URL

https://github.com/nvda-es/TeleNVDA/releases/download/2026.7.18/TeleNVDA-2026.7.18.nvda-addon

### Source URL

https://github.com/nvda-es/TeleNVDA

### Publisher

jmdaweb

### Channel

stable

### License Name

GPL v2

### License URL

https://www.gnu.org/licenses/gpl-2.0.html

**Comment by nvdaes** [comment]
Hello @jmdaweb. The download URL for this add-on is associated with nvda-es. Should nvda-es organization will be approved as a maintainer for this add-on, in addition to jmdaweb?

**Comment by jmdaweb** [comment]
Only if you see it necesary for the workflows to run properly. Currently I'm the only person updating this add-on on the store.

**Comment by nvdaes** [comment]
It shouldn't be necessary. I think thaqt it's better to have just jmdaweb as a maintainer, unless you prefer to get feedback from NV Access. You may close this issue and submit TeleNVDA hosted in your personal account.

**Comment by seanbudd** [comment]
@nvdaes nvda-es is an org, there's no need for @jmdaweb to rehost the add-on on his account. Organisations cannot post issues / submit add-ons etc. All we require is that an author has permission from the original publishers of the add-on. @jmdaweb is the original publisher of this add-on as part of NVDA-es. Even if someone is submitting from a fork they created, it doesn't mean we should accept the add-on. e.g. if Jim forks audioScreen, he is not the original publisher of it and shouldn't be able to submit without the current maintainers permission.

**Comment by seanbudd** [comment]
@nvdaes - @jmdaweb -is also already approved for this add-on, it was pending security review

**Comment by nvdaes** [comment]
@seanbudd wrote:

> nvda-es is an org, there's no need for @jmdaweb to rehost the add-on on his account. Organisations cannot post issues / submit add-ons etc. 

OK, I understand this well now. Sorry for the bad advice.
